### PR TITLE
[#35] Storage Balancing REP: Depend on iRODS 4.3.2

### PR DESCRIPTION
**Pull request by alanking**
Confirmed that the REP builds and works with the advanced training Rule Engine Plugins deck.

### CODE REVIEW

File: advanced/irods_rule_engine_plugin_storage_balancing/CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.12.0 FATAL_ERROR)
-find_package(IRODS 4.3.0 EXACT REQUIRED CONFIG)
+find_package(IRODS 4.3.2 EXACT REQUIRED CONFIG)
```
Comment (alanking) [inline advanced/irods_rule_engine_plugin_storage_balancing/CMakeLists.txt:2]: Alternatively, we can implement something like what we do with our newer versions so we don't have to keep touching this... https://github.com/irods/irods_rule_engine_plugin_audit_amqp/blob/2310a021680aa4142c5bc203e33a0dcce85a4ec6/CMakeLists.txt#L7-L10

Then again, we should probably just revamp this whole project eventually.

Comment (trel) [reply advanced/irods_rule_engine_plugin_storage_balancing/CMakeLists.txt:2]: yeah, either is fine for now - this is once per year at most...

Comment (alanking) [reply advanced/irods_rule_engine_plugin_storage_balancing/CMakeLists.txt:2]: Okay. Let's just leave it as is for now, then.

Comment (korydraughn) [reply advanced/irods_rule_engine_plugin_storage_balancing/CMakeLists.txt:2]: Opened #37. We can get that in post UGM.

**Comment by trel** [comment]
confirmed - just needs a # and merge it

**Comment by alanking** [comment]
#'d. Merging.

**Pull request merged**